pkg/apis/certmanager/v1/types_certificaterequest.go

```diff
@@ -131,14 +165,24 @@ type CertificateRequestSpec struct {
 // resulting signed certificate.
 type CertificateRequestStatus struct {
 	// List of status conditions to indicate the status of a CertificateRequest.
-	// Known condition types are `Ready` and `InvalidRequest`.
+	// Known condition types are `Ready`, `InvalidRequest`, `Approved` and `Denied`.
 	// +listType=map
 	// +listMapKey=type
 	// +optional
 	Conditions []CertificateRequestCondition `json:"conditions,omitempty"`
 
-	// The PEM encoded x509 certificate resulting from the certificate
+	// The PEM encoded x509 certificate chain resulting from the certificate
 	// signing request.
+	//
+	// This field must contain the leaf certificate followed by all intermediates
+	// that the client requires for verification.
+	//
+	// Root self-signed certificates SHOULD be omitted from the chain as per
+	// https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.2
+	// > [T]he self-signed certificate that specifies the root certificate authority
+	// > MAY be omitted from the chain, under the assumption that the remote end must
+	// > already possess it in order to validate it in any case.
```

Comment:
**question:** why is MAY being transformed into SHOULD?

Reply:
This is based on the comments here: https://github.com/cert-manager/cert-manager/blob/cab2b3b68ca834b60931fd76d1fb74f757a03550/pkg/util/pki/parse.go#L266-L270

Reply:
I'll move these changes to a seperate PR.